src/shared/extensionUtilities.ts

```diff
@@ -69,3 +69,19 @@ export function safeGet<O, T>(obj: O | undefined, getFn: (x: O) => T): T | undef
 
     return undefined
 }
+
+/**
+ * Utility function to search for tokens in a string and convert them to relative paths parseable by VS Code
+ * Useful for converting HTML images to webview-usable images
+ *
+ * @param basePath Extension path (from extension context)
+ * @param text Text to scan
+ * @param pattern Regex to search for (default: `/!!EXTENSIONROOT!!/g`)
+ */
+export function convertPathTokensToPath(
```

Comment:
The parameter ordering is not intuitive.
Follow a more conventional "replace" signature: convertXyz(inputText, pattern, replaceWith).

Examples: [JS](https://www.w3schools.com/jsref/jsref_replace.asp), [C#](https://docs.microsoft.com/en-us/dotnet/api/system.string.replace?view=netframework-4.8#System_String_Replace_System_String_System_String_)

Reply:
So the tricky part here is the default regex as a parameter...that only really works if that's in the final position.

Should I require that the caller provides this? At the very least, I can swap to `(inputText, replaceWith, pattern)`

Reply:
✔️ Function is now private and has the token hardcoded. Now fits the pattern of `inputText, replaceWith`